agg-ast/schema_derivation/src/schema_derivation_tests/match_expr.rs

```diff
@@ -1288,19 +1282,7 @@ mod date_ops {
                 "second": "$foo.second", 
                 "millisecond": "$foo.millisecond"
             }}}}"#,
-        ref_schema = Schema::Document(Document {
-            keys: map! {
-                "year".to_string() => Schema::Any,
-                "month".to_string() => Schema::Any,
-                "day".to_string() => Schema::Any,
-                "hour".to_string() => Schema::Any,
-                "minute".to_string() => Schema::Any,
-                "second".to_string() => Schema::Any,
-                "millisecond".to_string() => Schema::Any,
-            },
-            required: set!("year".to_string(), "month".to_string(), "day".to_string()),
-            ..Default::default()
-        })
+        ref_schema = Schema::Any
```

Comment:
Ha! Nice, that's a clever use of this in the context of testing.

Reply:
This is actually how I found out we should have this get_or_create function. This was the original version of this test and it didn't work :)